### ISSUE TITLE
Gear state and flaps state not changing

### ISSUE THREAD

**Issue by b7ackjokerr** [till213/SkyDolly#73]
Im Try on PMDG 737-800 and default msfs airbus a310.
Gear handle position went up but not the gear. It could be a MSFS issue.
i gave u record on yt with simvar watcher:

https://www.youtube.com/watch?v=SfEjIg99_Tw

my plugins:
-toolbar pushback
-nameplates

i use only Speedlink BLACK WIDOW XE Flightstick as a joystick.

**Comment by till213** [comment]
Hello,

Thanks for reporting! Yes, this is a known issue specifically with (3rd party) aircraft that use their own simulation logic, e.g. for hydraulics: they do not react to "simulation variables", as sent via SimConnect.

The good news: I have already started investigating other means to send those "state changes" (keyword: SimConnect "events"), and I have had quite some success already with the PDMG Boeing 737-800 (that I happen to own as well ;)).

Parts of it has just been merged into the "main" branch yesterday, but this is still "work in progress" and systematic testing with other aircraft such as the A310 remains to be done once all changes have fully been implemented.

Also refer to the corresponding announcement e.g. here, toward the end of this posting:

https://forums.flightsimulator.com/t/sky-dolly-v0-13-free-flight-recorder-replay-location-management/405367/58
